apps/gruenbuch/models.py

```diff
@@ -103,6 +103,25 @@ class GruenbuchOverviewPage(MetadataPageMixin, Page):
         ObjectList(de_ls_content_panels, heading='Easy German'),
     ])
 
+    subpage_types = ['apps_gruenbuch.GruenbuchDetailPage',
+                     'apps_gruenbuch.GruenbuchIndexPage']
+
+    search_fields = Page.search_fields + [
+        index.SearchField('page_title_de', partial_match=True),
+        index.SearchField('page_title_en', partial_match=True),
+        index.SearchField('page_title_de_ls', partial_match=True),
+        index.SearchField('page_intro_de', partial_match=True),
+        index.SearchField('page_intro_en', partial_match=True),
+        index.SearchField('page_intro_de_ls', partial_match=True),
```

Comment:
I assume each page model needs it's own search field even if it extends another model right? @goapunk